EmployeeList/Persistence/OnDiskCache.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Cache.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Combine
+
+final class Cache {
+    
+    private var fileManager: FileManager = FileManager.default
```

Comment:
Why have this? If you do, you don't need to type it:

`private let fileManager = FileManager.default`

Reply:
A general question: Do we ignore explicit typing when we initialize a variable with an actual value in Swift?

Reply:
the variable contents set the type of the var. Here the contents are a FileManager, so the var is statically typed to that.